styles.css

```diff
@@ -0,0 +1,80 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  padding: 30px 20px;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  margin: 0;
+  justify-content: space-between;
+  color: white;
+  background-color: #3c3a39;
+  text-decoration: none;
+  padding: 20px;
+}
+
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.logo {
+  color: white;
+  text-decoration: none;
+}
+
+.heading {
+  color: #ff6b00;
+  font-size: 50px;
+  font-family: Crete Round, var(--font-family-common);
+  margin: 10% 0 0;
+}
+
+.describe {
+  font-family: Roboto, var(--font-family-common);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  color: #fff;
+}
+
+.headline {
+  font-family: Poppins, var(--font-family-common);
+  font-size: 20px;
+  line-height: 32px;
+  color: #fff;
+}
+
+/* This is the social styling */
+
+.social {
+  display: flex;
+}
```

Comment:
- [OPTIONAL] Kindly make sure to remove these dots which is appearing on all unordered list and please make sure to center your social media icons in order to meet the provided Figma design. :wink: